jobs/broken-site-report-ml/broken_site_report_ml/main.py

```diff
@@ -213,8 +213,10 @@ def main(bq_project_id, bq_dataset_id):
     except Exception as e:
         logging.error(e)
         is_ok = False
+        raise
 
-    record_classification_run(client, bq_dataset_id, is_ok, result_count)
+    finally:
+        record_classification_run(client, bq_dataset_id, is_ok, result_count)
 
 
```

Comment:
I modified error handling here to make sure the job fails if any of requests to bugbug fail.